docs/book/src/getting-started/testdata/project/Makefile

```diff
@@ -78,12 +78,16 @@ test-e2e: manifests generate fmt vet ## Run the e2e tests. Expected an isolated
 	}
 	go test ./test/e2e/ -v -ginkgo.v
 
+.PHONY: lint-config
+lint-config: golangci-lint ## Verify golangci-lint linter configuration
+	$(GOLANGCI_LINT) config verify
+
 .PHONY: lint
-lint: golangci-lint ## Run golangci-lint linter
+lint: lint-config ## Run golangci-lint linter
```

Comment:
I think we should not call the config
When I am running make lint I want lint the code not validate the lint config

Reply:
My line of thinking was that if I have a invalid config, I should error instead of running the linter with it

But I can change

what do you prefer?

Reply:
done